eosc/cmd/voteStatus.go

```diff
@@ -0,0 +1,66 @@
+package cmd
+
+import (
+	"fmt"
+
+	eos "github.com/eoscanada/eos-go"
+	"github.com/spf13/cobra"
+)
+
+var voteStatusCmd = &cobra.Command{
+	Use:   "status [voter name]",
+	Short: "Display the current vote status for a given account.",
+	Args:  cobra.MinimumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		api := getAPI()
+		voterName := toAccount(args[0], "voter name")
+
+		response, err := api.GetTableRows(
+			eos.GetTableRowsRequest{
+				Code:       "eosio",
+				Scope:      "eosio",
+				Table:      "voters",
+				JSON:       true,
+				LowerBound: string(voterName),
+				Limit:      1,
+			},
+		)
+		errorCheck("get table row", err)
+
+		var voterInfos []eos.VoterInfo
+		err = response.JSONToStructs(&voterInfos)
+		errorCheck("reading voter_info", err)
+
+		found := false
+		for _, info := range voterInfos {
```

Comment:
Not blocking for merge, but we should probably extract the logic on finding the voter in the voters list in a helper as it's shared across multiple files now.